azurerm/helpers/azure/sender.go

```diff
@@ -0,0 +1,57 @@
+package azure
+
+import (
+	"log"
+	"net/http"
+	"net/http/httputil"
+
+	"github.com/Azure/go-autorest/autorest"
+)
+
+func BuildSender() autorest.Sender {
```

Comment:
Nit: Exported functions should be documented

Reply:
done 👍